app/konnect/runtime-manager/consumer-groups.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Create Consumer Groups in Konnect
+content_type: tutorial
+badge: enterprise
+---
+With consumer groups, you can define any number of rate limiting tiers and
+apply them to subsets of consumers, instead of managing each consumer
+individually.
+
+For example, you could define three consumer groups:
+* A "gold tier" with 1000 requests per minute
+* A "silver tier" with 10 requests per second
+* A "bronze tier" with 6 requests per 30 seconds
+
+The `consumer_groups` endpoint works together with the [Rate Limiting Advanced plugin](/hub/kong-inc/rate-limiting-advanced).
+
+Consumers that are not in a consumer group default to the Rate Limiting advanced
+plugin’s configuration, so you can define tier groups for some users and
+have a default behavior for consumers without groups.
+
+To use consumer groups for rate limiting, you must:
+* Create one or more consumer groups
+* Create consumers
```

Comment:
Can we merge these steps? Like "Create consumers and add them to an already exisisting consumer group", or "Create a consumer group and create or add consumers to it"